Makefile

```diff
@@ -143,15 +126,15 @@ test-unit-coverage: ##@tests Run unit and integration tests with coverage
 test-e2e: ##@tests Run e2e tests
 	# order: reliability then alphabetical
 	# TODO(tiabc): make a single command out of them adding `-p 1` flag.
-	go test -timeout 5m ./t/e2e/accounts/... -network=$(networkid)
-	go test -timeout 5m ./t/e2e/api/... -network=$(networkid)
-	go test -timeout 5m ./t/e2e/node/... -network=$(networkid)
-	go test -timeout 50m ./t/e2e/jail/... -network=$(networkid)
-	go test -timeout 20m ./t/e2e/rpc/... -network=$(networkid)
-	go test -timeout 20m ./t/e2e/whisper/... -network=$(networkid)
-	go test -timeout 10m ./t/e2e/transactions/... -network=$(networkid)
+	go test -timeout 5m ./t/e2e/accounts/... -network=$(or $(networkid),StatusChain)
```

Comment:
We can set it like this to have a default value:
```
GO ?= StatusChain
```

Reply:
Oh, nice, more elegant.